aem/aem_skyline_install.md

```diff
@@ -0,0 +1,98 @@
+
+## Integrate with AEM as a Cloud Service
+
+This documentation is specific to `AEM as a Cloud Service` set up, to integrate with `AEM on premise` 
+please refer to the other associated [documentation](aem_on_premise_install.md).
+
+### Pre-release provisioning
+
+If you are an existing `AEM as a Cloud Service` customer, please contact your Adobe representative providing your AEM authors public urls.
+
+For that, log-in to [your cloud manager](my.cloudmanager.adobe.com/) space, look up your authors environments
+
+  ![Cloud Manager Environments](../img/cloud_manager_environments.png "Cloud Manager Environments")
+
+Select from this list the environments public url you want Adobe I/O to expose as `Events Providers` . 
+
+As soon as the Adobe I/O team is done provisioning it, your new `AEM as a Cloud Service` `Events Providers` should appear in the Adobe I/O console.
+ 
+  ![An Adobe I/O Console `AEM as a Cloud Service` Events Provider](../img/add_skyline_event_provider.png "An Adobe I/O Console `AEM as a Cloud Service` Events Provider")
+ 
+You are now ready to set up your `AEM as a Cloud Service` environments for them to start emitting events, see below.
+
+### Deploy Adobe I/O events integration using Cloud Manager
+
+As this Adobe I/O events integration is not yet part of the core product, you will have to
+
+1. [Download the code from this repository](https://github.com/AdobeDocs/adobeio-events/releases/download/2020_07_20_13_00/aem-event-proxy-skyline-2020_07_20_13_00.zip),
+2. Uncompress this archive, and merge it with your own existing Cloud Manager git repo,
+see [Experience Manager Cloud Manager - Help - Accessing Git ](https://docs.adobe.com/content/help/en/experience-manager-cloud-manager/using/managing-code/accessing-git.html) 
+for further instructions,
+3. [Deploy it](https://docs.adobe.com/content/help/en/experience-manager-cloud-manager/using/how-to-use/deploying-code.html) 
+
+
+### Configure Adobe I/O authentication
+
+To secure the calls between Adobe I/O and AEM, we leverage an oAuth JWT exchange token flow.
+This flow uses a certificate to sign the JWT request and therefore requires certificates configurations
+on both ends. Please look at our documentations:
+* first [set up a keystore on AEM](aem_keystore_setup.md) 
+* then [set up an AEM workspace in Adobe I/O console](aem_console_setup.md)
+
+Once these 2 setups are done, you should have:
+
+* created a public/private certificate key and an associated keystore
+* added the keystore into the AEM `eventproxy-service` user&rsquo;s keystores vault.
+* created an Adobe I/O Console project and workspace using the public certificate
+
+You are now ready to finalize the Adobe IMS configuration needed by Adobe I/O Events:
+
+1. Open the Cloud Services console, or select the **Security** icon, and then select **Adobe IMS Configurations**.
```

Comment:
Cloud Services console?

Reply:
good catch it's the `Tools` section not the `Cloud Services`